src/agent/vm.js

```diff
@@ -524,6 +524,16 @@ var vm = {
       }
     });
   },
+
+  _getVMHostonlyInterface(vm_name) { //FIXME: use this here
+    return VM.info(vm_name).then((info) => {
+      if (info.installed) {
+        return info.hostonlyadapter1;
+      }
+      return null;
+    });
+  },
+
```

Comment:
This probably will be a public method to be used in configure, then it must be without the _.